ChangeLog

```diff
@@ -1,5 +1,113 @@
 ### 3.10.0dev <- NOTE: the release version number will be 3.11.0 ###
 
+- Refactor src/main.cpp (#3191).
+  (contributed by @pljones)
+
+- Refactor some `CHostAddress` parameters to be const references (#3259).
+  (contributed by @softins)
+
+- Client/Server: Improved start up logging (#3117).
+  (contributed by @pljones)
+
+- Client: Added "delete server" button to connect dialog (#3159).
+  (contributed by @pljones)
+
+- Client/Server: Display Qt versions for both build and runtime (#3231).
+  (contributed by @softins)
+
+- Client: Added "Delete Entry" button to Advanced Settings, Custom Directories (#3260).
+  (contributed by @AdamGLIN)
+
+- Client: Fixed the display of delete buttons on Mac (#3305).
+  (contributed by @softins)
+
+- Server: Fix systemd ExecStart command line to support start/stop recording signal (#3187).
+  (contributed by @rdica)
+
+- Server: Only de-register on quit (#3302).
+  (contributed by @pljones)
+
+- Android: save settings on app state change (#3144).
+  (contributed by @pljones)
+
+- Translations updated from Hosted Weblate (#3193).
+  (contributed by @weblate)
+
+- Translations updated from Hosted Weblate (#3246).
+  (contributed by @weblate)
+
+- Documentation: Improved link for Server CLI options (#3303).
+  (contributed by @gilgongo)
+
+- Documentation: updated TRANSLATING.md for current build process (#3312).
+  (contributed by @softins)
+
+- Build: adds macOS signed pkg build automation (#2624).
+  (contributed by @danryu)
+
+- Build: Bump actions/checkout from 3 to 4 (#3168).
+  (contributed by @app/dependabot)
+
+- Build: Bump maxim-lobanov/setup-xcode from 1.5.1 to 1.6.0 (#3181).
+  (contributed by @app/dependabot)
+
+- Build: Updated bundled Qt6 to version 6.6.0 (#3182).
+  (contributed by @app/github-actions)
+
+- Build: Ensure apt version >=2.4 to prove that the system is Debian based (using apt and dpkg) and can validate the repo (#3199).
```

Comment:
Hm.  I edited this one in the PR, too...  Not sure why it's still got the old text.  Probably a " C H A N G E L O G " tag in a comment I missed.